### ISSUE TITLE
[Bug]: No organizationURL given

### ISSUE THREAD

**Issue by 24mu13** [clemlesne/blue-agent#242]
Even if it's present on **values.yaml**, when running on the KEDA operator, I get the following error:

```
2024-06-04T18:58:57Z ERROR Failed to ensure ScaledJob is correctly created {"controller": "scaledjob", "controllerGroup": "keda.sh", "controllerKind": "ScaledJob", "ScaledJob": {"name":"agent-blue-agent","namespace":"default"}, "namespace": "default", "name": "agent-blue-agent", "reconcileID": "c4a72bcb-4c9a-420d-b3f3-7d8527e34ba0", "error": "error parsing azure Pipelines metadata: no organizationURL given"}
```

The error to the user in CLI is the following instead:
```
* Pod "agent-blue-agent-1" is invalid: spec.containers[0].lifecycle.preStop.exec.command: Required value
```

**Comment by Riqardos** [comment]
@24mu13 I had the same problem, issue is with these parameters:
```
 "pipelines" : {
      "cache" : {
        "volumeEnabled": true
      }
      "tmpdir": {
        "volumeEnabled": true
      }
    }
```

if they are both set to true, no command is set to `azp-agent` container and it ends up with error above.

https://github.com/clemlesne/blue-agent/blob/66d82a26837490803fab6548f011e702be32d6c1/src/helm/blue-agent/templates/_helpers.tpl#L167

The solution would be to add some dummy command there in the helm template or make the preStop hook customizable.

**Comment by Riqardos** [comment]
created [PR](https://github.com/clemlesne/blue-agent/pull/246) for this

**Comment by clemlesne** [comment]
PR raquires a few fixes, it should be done quickly.

Thank you @24mu13 for the bug report!

**Comment by clemlesne** [comment]
Solved by #267.

Please re-open if it still causes you issues.

**Comment by clemlesne** [comment]
Fix published in the `v7.1.2`.
